README.md

```diff
@@ -1,2 +1,202 @@
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 The Blog Ruby on Rails App <a name="about-project"></a>
+
+**XpensTrackr** is a budget management application built with Ruby-on-Rails.
+It is my Capstone project for the RoR module
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/HTML">HTML</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/CSS">CSS</a></li>
+    <li><a href="https://sass-lang.com/">SASS</a></li>
+    <li><a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript">JavaScript</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/">Ruby</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Built using MVC**
+- **Authentication with Devise**
+- **Authurization with Cancancan**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo](https://xpenstrackr.onrender.com/)
+
+<!-- VIDEO DEMO -->
+## Video Demo <a name="live-demo"></a>
+
+- [Video Demo](https://www.loom.com/share/66e7963e9f3f4b0885be5ec092a46b21)
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need the followings installed and running on your system:
+
+- Git
+- Ruby
+- Rails
+- PostgreSQL
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<pre>git clone git@github.com:kkolade/rails_blog_app.git</pre>
+
+- `cd xpenstrackr`
+- `code .` <!-- If you are using VS Code -->
+<!--
+Example commands:
+--->
+
+### Install
+
+Install the gems and packges required for this project with:
+
+```sh
+  gem install
+  npm install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<pre>rails s</pre>
+
+Open your browser and go to:
+<pre>http://localhost:3000/</pre>
+
+### Run tests
+
+To run tests, run the following command:
+
+<pre>rspec ./spec/</pre>
+
```

Comment:
- Hey, while you've made great effort in this project, please, let's kindly add some unit test as the current version of your project is not having any test added to it. kindly do this to meet with the project requirement as seen below:


![image](https://github.com/kkolade/XpensTrackr/assets/59397159/d6640023-ba00-4ea6-88af-4ff137e19beb)